### ISSUE TITLE
Bootloader Unlock

### ISSUE THREAD

**Issue by quaizaco** [kobolabs/Kobo-Arc#2]
Hi everybody !
I tried to install a custom rom on my Kobo Arc and I think that I made a wrong manip so now I can't access to the ADB mode. I just can access to the fastboot mode but I can't fix my problem because there is a locked bootloader on the Kobo Arc so I would like to know If someone can help me to unlock the bootloader of the kobo arc ?

**Comment by quaizaco** [comment]
No, when I try to flash something with the fastboot, It tells me "remote failed". And can you tell me where can I find the right clockwork recovery.img for the kobo Arc ?

**Comment by kobocar** [comment]
can someone please give the stock android 4.2.2 rom for the kobo arc 7 or the recovery images
